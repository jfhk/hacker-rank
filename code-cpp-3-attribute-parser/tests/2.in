4 8
<tag1 value = "HelloWorld" okay = "what">
<tag2 name = "Name1">
</tag2>
</tag1>
tag1.tag2~name
tag1~name
tag1~value
tag2~name
tag2~what
tag1~okay
tag5~value
tag1.tag5~name
