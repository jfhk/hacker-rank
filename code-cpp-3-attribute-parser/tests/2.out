Name1
Not Found!
HelloWorld
Not Found!
Not Found!
what
Not Found!
Not Found!
